### ISSUE TITLE
mantine Pagination buttons lacking labels for accessibility

### ISSUE THREAD

**Issue by ocshawn** [mantinedev/mantine#7551]
### Dependencies check up

- [x] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.17.1

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

All

### Describe the bug

Buttons must have discernible text for accessibility

### If possible, include a link to a codesandbox with a minimal reproduction

https://codesandbox.io/p/sandbox/mantine-react-template-forked-xpncrt

### Possible fix

Previous and Next buttons could use an aria-label 

### Self-service

- [x] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
It is intended behavior, Mantine does not include localization features, you must add those labels on your side using `getItemProps` and `getControlProps` props.

**Comment by giammyisjammy** [comment]
@ocshawn I had the same need, ended up defining a custom [default props](https://mantine.dev/theming/default-props/) for `Pagination` in my `theme.components` where I set the proper attributes.

Like this:

```tsx
// src/theme.ts

import { createTheme, Pagination } from '@mantine/core';

import { i18next } from './i18n';

export const theme = createTheme({
  components: {
    Pagination: Pagination.extend({
      defaultProps: {
        getItemProps: (page) => ({
          title: i18next.t('pagination.page', { page }),
        }),
        getControlProps: (control) => ({
          title: i18next.t(`pagination.${control}`),
        }),
      },
    }),
  },
});
```

Keep in mind that I made this snippet using `react-i18next` so YMMV.
